packages/lib/i18n.ts

```diff
@@ -12,3 +12,14 @@ export const localeOptions = locales.map((locale) => ({
 export const defaultLocaleOption = localeOptions.find(
   (locale) => locale.value === i18n.defaultLocale
 ) as (typeof localeOptions)[number];
+
+// List of known RTL locales, it returnbs rtl or ltr for respective locales
+const rtlLocales = ["ar", "he"];
+
+export function getTextDirection(locale: string): "ltr" | "rtl" {
+  return rtlLocales.includes(locale) || rtlLocales.includes(locale.split("-")[0]) ? "rtl" : "ltr";
+}
+
+export function isRTL(locale: string): boolean {
```

Comment:
Also coming to think of it, isRTL is a very good function name, you can't tell what it does. Please consider renaming it to something more descriptive.

```suggestion
export function isLocaleRightToLeft(locale: string): boolean {
```

Or something

Reply:
I'll consider it haha, I just thought it would make sense to name it as rtl for "right to left" 

rn I'm away from home for a while so can't access the laptop but I'll try to do it soon

Reply:
Yeah no worries. 
It makes sense now because we know the context. Someone not knowing would have difficulty understanding what `isRTL` means